test/units/galaxy/test_token.py

```diff
@@ -6,16 +6,110 @@
+    )
+
+
+@pytest.fixture()
+def public_key_numbers():
+    return {
+        "n": (
+            "ofgWCuLjybRlzo0tZWJjNiuSfb4p4fAkd_wWJcyQoTbji9k0l8W26mPddxHmfHQp-"
+            "Vaw-4qPCJrcS2mJPMEzP1Pt0Bm4d4QlL-yRT-SFd2lZS-pCgNMsD1W_YpRPEwOWvG"
+            "6b32690r2jZ47soMZo9wGzjb_7OMg0LOL-bSf63kpaSHSXndS5z5rexMdbBYUsLA9"
+            "e-KXBdQOS-UTo7WTBEMa2R2CapHg665xsmtdVMTBQY4uDZlxvb3qCo5ZwKh9kG4LT"
+            "6_I5IhlJH7aGhyxXFvUK-DWNmoudF8NAco9_h9iaGNj8q2ethFkMLs91kzk2PAcDT"
+            "W9gb54h4FRWyuXpoQ"
+        ),
+        "e": "AQAB"
+    }
+
+
+@pytest.fixture()
+def private_key(public_key_numbers):
+    private_key_data = {
+        "kty": "RSA",
+        "d": (
+            "Eq5xpGnNCivDflJsRQBXHx1hdR1k6Ulwe2JZD50LpXyWPEAeP88vLNO97IjlA7_GQ"
+            "5sLKMgvfTeXZx9SE-7YwVol2NXOoAJe46sui395IW_GO-pWJ1O0BkTGoVEn2bKVRU"
+            "Cgu-GjBVaYLU6f3l9kJfFNS3E0QbVdxzubSu3Mkqzjkn439X0M_V51gfpRLI9JYan"
+            "rC4D4qAdGcopV_0ZHHzQlBjudU2QvXt4ehNYTCBr6XCLQUShb1juUO1ZdiYoFaFQT"
+            "5Tw8bGUl_x_jTj3ccPDVZFD9pIuhLhBOneufuBiB4cS98l2SR_RQyGWSeWjnczT0Q"
+            "U91p1DhOVRuOopznQ"
+        ),
+        "p": (
+            "4BzEEOtIpmVdVEZNCqS7baC4crd0pqnRH_5IB3jw3bcxGn6QLvnEtfdUdiYrqBdss"
+            "1l58BQ3KhooKeQTa9AB0Hw_Py5PJdTJNPY8cQn7ouZ2KKDcmnPGBY5t7yLc1QlQ5x"
+            "HdwW1VhvKn-nXqhJTBgIPgtldC-KDV5z-y2XDwGUc"
+        ),
+        "q": (
+            "uQPEfgmVtjL0Uyyx88GZFF1fOunH3-7cepKmtH4pxhtCoHqpWmT8YAmZxaewHgHAj"
+            "LYsp1ZSe7zFYHj7C6ul7TjeLQeZD_YwD66t62wDmpe_HlB-TnBA-njbglfIsRLtXl"
+            "nDzQkv5dTltRJ11BKBBypeeF6689rjcJIDEz9RWdc"
+        ),
+        "dp": (
+            "BwKfV3Akq5_MFZDFZCnW-wzl-CCo83WoZvnLQwCTeDv8uzluRSnm71I3QCLdhrqE2"
+            "e9YkxvuxdBfpT_PI7Yz-FOKnu1R6HsJeDCjn12Sk3vmAktV2zb34MCdy7cpdTh_YV"
+            "r7tss2u6vneTwrA86rZtu5Mbr1C1XsmvkxHQAdYo0"
+        ),
+        "dq": (
+            "h_96-mK1R_7glhsum81dZxjTnYynPbZpHziZjeeHcXYsXaaMwkOlODsWa7I9xXDoR"
+            "wbKgB719rrmI2oKr6N3Do9U0ajaHF-NKJnwgjMd2w9cjz3_-kyNlxAr2v4IKhGNpm"
+            "M5iIgOS1VZnOZ68m6_pbLBSp3nssTdlqvd0tIiTHU"
+        ),
+        "qi": (
+            "IYd7DHOhrWvxkwPQsRM2tOgrjbcrfvtQJipd-DlcxyVuuM9sQLdgjVk2oy26F0Emp"
+            "ScGLq2MowX7fhd_QJQ3ydy5cY7YIBi87w93IKLEdfnbJtoOPLUW0ITrJReOgo1cq9"
+            "SbsxYawBgfp_gh6A5603k2-ZQwVK0JKSHuLFkuQ3U"
+        ),
+    }
+    private_key_data.update(public_key_numbers)
+
+    e = b64url_to_long(private_key_data['e'])
+    n = b64url_to_long(private_key_data['n'])
+    public_numbers = RSAPublicNumbers(e, n)
+
+    p = b64url_to_long(private_key_data['p'])
+    q = b64url_to_long(private_key_data['q'])
+    d = b64url_to_long(private_key_data['d'])
+    dmp1 = b64url_to_long(private_key_data['dp'])
+    dmq1 = b64url_to_long(private_key_data['dq'])
+    iqmp = b64url_to_long(private_key_data['qi'])
+
+    private_numbers = RSAPrivateNumbers(p, q, d, dmp1, dmq1, iqmp, public_numbers)
+    yield private_numbers.private_key()
```

Comment:
There's no need for a generator unless you do some clean-up post-`yield`.
```suggestion
    return private_numbers.private_key()
```